server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java

```diff
@@ -56,20 +57,20 @@ private boolean indexesAreEnabled() {
   private void indexEmptyTypes(StartupIndexer indexer) {
     Set<IndexType> emptyTypes = getEmptyTypes(indexer);
     if (!emptyTypes.isEmpty()) {
-      log(indexer, emptyTypes, "...");
+      Profiler profiler = Profiler.create(LOG);
+	  profiler.startInfo(getLogMessage(emptyTypes, "..."));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)